pages/ohjeita-en.md

```diff
@@ -34,6 +34,9 @@ New api versions will be announced in Digitraffic website and Google-groups
 [marine](https://groups.google.com/g/meridigitrafficfi) and
 [rail](https://groups.google.com/u/1/g/rata_digitraffic_fi).
 
+Deprecated APIs and their sunset dates are updated [on this page](/rajapintojen-tila/muutokset/).
```

Comment:
Musta on kivempi, jos `on this page` tilalla lukee se polku mistä se löytyy, eli `Support → API changes`